app/gazebo/tug/plugins/tuginterface/interface/CMakeLists.txt

```diff
@@ -6,14 +6,14 @@ set(gazebo_assistiverehab_tuginterface_source src/tuginterface.cpp src/tugserver
 set(gazebo_assistiverehab_tuginterface_headers include/tuginterface.h include/tugserver.h include/utils.h)
 
 include_directories(${CMAKE_CURRENT_SOURCE_DIR}
```

Comment:
@vvasco you may consider using `target_include_directories` instead, which is newer.